autometa/taxonomy/ncbi.py

```diff
@@ -434,20 +434,17 @@ def convert_taxid_dtype(self, taxid):
         if taxid <= 0:
             raise ValueError(f"Taxid must be a positive integer! Given: {taxid}")
         # Checking databases
-        if taxid not in self.names and taxid not in self.nodes and taxid in self.merged:
-            taxid = self.merged[taxid]
-            if taxid not in self.names and taxid not in self.nodes:
-                err_message = f"NCBI databases out of sync. Merged taxid ({taxid}) not found in nodes.dmp or names.dmp!"
+        if taxid not in self.names and taxid not in self.nodes:
+            if taxid not in self.merged:
+                err_message = f"NCBI databases out of sync. {taxid} not in found in any of the NCBI databases - nodes.dmp, names.dmp and merged.dmp."
```

Comment:
Can we make this more concise?

```suggestion
                err_message = f"Databases out of sync. {taxid} not in found in nodes.dmp, names.dmp or merged.dmp."
```